src/Integration.UnitTests/LocalServices/ProjectSystemHelperTests.cs

```diff
@@ -93,16 +94,16 @@ public void ProjectSystemHelper_GetSolutionProjects_ReturnsOnlyKnownLanguages()
         {
             // Arrange
             ProjectMock csProject = this.solutionMock.AddOrGetProject("c#");
-            csProject.SetProperty(VSConstants.VSITEMID_ROOT, (int)__VSHPROPID.VSHPROPID_ExtObject, csProject);
+            csProject.SetExtObjProperty(VSConstants.VSITEMID_ROOT, csProject);
```

Comment:
Is SetExtObjProperty used for anything else?
We can move it to ProjectMock's ctor, to always set VSITEMID_ROOT to point to `this`